stream_chat/client.py

```diff
@@ -267,6 +267,33 @@ def get_devices(self, user_id):
         """
         return self.get("devices", {"user_id": user_id})
 
+    def get_rate_limits(
```

Comment:
we need to base definition in `base/client.py` and also async implementation in `async_chat/client.py`